extensions/wikia/ImageServing/drivers/ImageServingDriverUserNS.class.php

```diff
@@ -17,11 +17,9 @@ function formatResult( $allImages, $filteredImages ) {
 	}
 
 	private function getAvatar( $user ) {
-		$cut = $this->imageServing->getCut( 100, 100, "center", false );
-
 		return array(
 			"name" => 'avatar',
-			"url" => AvatarService::getAvatarUrl( $user, $cut )
+			"url" => AvatarService::getAvatarUrl( $user, 200 )
```

Comment:
I think I'd use something like `max( $this->minWidth, $this->minHeight, 200)`, since these might request something bigger.

Reply:
But besides that, LGTM.